lib/influxer/metrics/metrics.rb

```diff
@@ -122,13 +121,18 @@ def quote(name)
       # rubocop:enable Metrics/MethodLength
     end
 
-    delegate :tag_names, to: :class
+    delegate :tag_names, to: 'self.class'
```

Comment:
Let's get rid of delegation in favour of explicit method:

```ruby
def tag_names
  self.class.tag_names
end
```

That would not depend on Rails at all.